discovery-provider/tests/conftest.py

```diff
@@ -202,3 +204,25 @@ def get_db_read_replica():
     )
 
     return mock
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Do we need this? I'm pretty confident we drop the tables after every test already